src/main/java/org/example/server/VerificationServer.java

```diff
@@ -0,0 +1,126 @@
+package org.example.server;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.net.SocketTimeoutException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+public class VerificationServer {
+    private static final String password = "пиріг";
+    private static final String client = "Кліент: ";
+    private static final int port = 8081;
```

Comment:
you already declared this var once, why do you need to declare it a second time?